pandas/tests/tools/test_to_datetime.py

```diff
@@ -383,6 +383,26 @@ def test_to_datetime_parse_timezone_keeps_name(self):
         expected = DatetimeIndex(["2010-01-01 12:00:00"], tz="UTC", name="foo")
         tm.assert_index_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "data, format, expected",
+        [
+            ([pd.NA], "%Y%m%d%H%M%S", DatetimeIndex(["NaT"])),
```

Comment:
can you add another test with a valid element & pd.NA, also one with None, np.nan & pd.NA

Reply:
added